PRINCIPLES.md

```diff
@@ -12,7 +12,15 @@ Last reviewed and concurred: Jul 27, 2022, [Issue 413](https://github.com/cncf/t
 [No Kingmakers & One Size Does Not Fit All](#no-kingmakers--one-size-does-not-fit-all)  
 [Not a Standards Body](#not-a-standards-body)  
 [We Want a Comprehensive Toolchain](#we-want-a-comprehensive-toolchain)    
-[Above All We Want To Help Projects](#above-all-we-want-to-help-projects)  
+[Above All We Want To Help Projects](#above-all-we-want-to-help-projects)
+
+
+[Technical Leadership Principles](#technical-leadership-principles)    
+[Purpose](#purpose)    
+[Be welcoming](#be-welcoming)    
```

Comment:
Maybe under "Be welcoming" a section could be added for "train the next leaders?"  Identifying individuals who are helpful to the project/community consistently and helping them take over roles.  Give opportunities for others to help, if people are willing, and give them support.  Help them identify what roles do suit them and follow interests, and make sure not to ask for help just because someone is good at something but also because that is their interest.  Sometimes people get slotted into roles they have no interest in, like organizing or event planning, when really they are there for technical content, and we should be sensitive to that.